src/theme/globalStyles.ts

```diff
@@ -0,0 +1,65 @@
+import { createGlobalStyle } from "styled-components";
+
+const GlobalStyle = createGlobalStyle`
+	body {
+		font-family: ${({ theme: { fonts } }) => fonts.primaryFont};
+		background: ${({ theme }) => theme.colors.primaryGreen};
+		font-size: ${(props) => props.theme.fontSizes.regularParagraph}px;
+	}
+
+	button {
+		font-family: ${({ theme: { fonts } }) => fonts.primaryFont};
```

Comment:
do you really need to specify the font-family in various places? It's always the primary one. If defining it in `body` doesn't work, try `html` tag.